### ISSUE TITLE
get_api() output not consistent

### ISSUE THREAD

**Issue by testerbb** [studioimaginaire/phue#124]
Hi everybody

Calling twice the get_api()-function prints twice different results.
Furthermore, non of the python outputs corresponds to the hue output which can be retrieved by the hue/html api.
It seems the problem is related to some buffering because the python-outputs are extracts from the hue-output.
Does anybody have experience with this problem? 
thanks in advance for the help

Best regards

**Comment by natcl** [comment]
Can you send example prints ?

**Comment by testerbb** [comment]
This is the HTML response from hue-api

![hue api 2018-01-02_18-17-49](https://user-images.githubusercontent.com/35032276/34493541-b290cafe-efec-11e7-9c4e-02b03da8fa89.png)

And these are screenshots from the python output. The code is very short and simple. Nevertheless, the output is not the same. 

![2018-01-02-181833_1824x984_scrot](https://user-images.githubusercontent.com/35032276/34493572-d943e01e-efec-11e7-8486-872541c4617b.png)
![2018-01-02-182926_1824x984_scrot](https://user-images.githubusercontent.com/35032276/34493573-d9643300-efec-11e7-97ae-faaf3a887a26.png)
![2018-01-02-183053_1824x984_scrot](https://user-images.githubusercontent.com/35032276/34493574-d983b306-efec-11e7-8d37-937240401023.png)

**Comment by natcl** [comment]
Are you sure it's different ? The output of get_api() is not "pretty printed" by default. That's probably the only difference...

**Comment by testerbb** [comment]
Yes, the starting is not always the same. Once the output starts with "schedules", once "light" and another time with "config"

**Comment by natcl** [comment]
This is normal, there is not guarantee of order in JSON/Python
dictionaries.  As long as all the data is there it is fine.

2018-01-02 13:52 GMT-05:00 testerbb <notifications@github.com>:

**Comment by testerbb** [comment]
Thanks for the hint.
Nevertheless, on hue html api the structure is always the same, but probably its written in java and not in python. I will check the structures and come back to it

Thanks
